index.js

```diff
@@ -24,6 +24,7 @@ export default class QRCodeScanner extends Component {
     reactivateTimeout: PropTypes.number,
     fadeIn: PropTypes.bool,
     showMarker: PropTypes.bool,
+    camType: PropTypes.string,
```

Comment:
can we make this more explicit using `PropTypes.oneOf(['back', 'front'])`

Reply:
also can we rename this to `cameraType`, clearer if variable names aren't shortened.